spec/fbp.coffee

```diff
@@ -136,6 +226,28 @@ describe 'FBP parser', ->
         chai.expect(graphData.inports).to.be.an 'undefined'
         chai.expect(graphData.outports).to.be.an 'undefined'
 
+  describe 'with FBP string containing a JSON IIP string', ->
```

Comment:
Sorry I forgot about this. It should be fine since i'm using peg.js' own JSON grammar, which I assume has been thoroughly tested, but I can do a followup PR to add a test if you want.